cfme/common/__init__.py

```diff
@@ -75,13 +75,20 @@ def _assign_unassign_policy_profiles(self, assign, *policy_profile_names):
             policy_profile_names: :py:class:`str` with Policy Profile names.
         """
         view = navigate_to(self, 'ManagePoliciesFromDetails')
+        policy_changed = False
         for policy_profile in policy_profile_names:
             if assign:
-
-                view.policy_profiles.check_node(policy_profile)
+                policy_changed = view.policy_profiles.fill(
+                    view.policy_profiles.CheckNode([policy_profile])
+                ) or policy_changed
```

Comment:
**Required**

fill will return boolean, so there's no need for an `or` statement here, or setting of `policy_changed` before the fill.

**Optional**
Collapse this a bit - haven't explicitly tested, but an example of the more concise syntax:
```python
changed = view.policy_profiles.fill(
    getattr(view.policy_profiles, 'CheckNode' if assign else 'UncheckNode')([policy_profile])
)